### PR TITLE
Automation fix for GCE UI based on feature changes

### PR DESCRIPTION
**Pull request by adarshdubey-star**
`Changes needed for the New GCE CR page `
For better results run with https://github.com/SatelliteQE/airgun/pull/807

Signed-off-by: Adarsh Dubey <addubey@redhat.com>

### CODE REVIEW

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_computeresource_gce.py
airgun: 807

File: tests/foreman/ui/test_computeresource_gce.py [hunk 1]
```diff
@@ -87,8 +89,6 @@ def test_positive_default_end_to_end_with_custom_profile(
         assert cr_values['provider_content']['http_proxy']['value'] == http_proxy.name
         assert cr_values['organizations']['resources']['assigned'] == [module_org.name]
         assert cr_values['locations']['resources']['assigned'] == [module_location.name]
```
Comment (shweta83) [inline tests/foreman/ui/test_computeresource_gce.py:91]: You can assert json_key here.

**Review by shweta83 — CHANGES_REQUESTED** [review]

File: tests/foreman/ui/test_computeresource_gce.py (hunk 1 above)
Comment (adarshdubey-star) [reply tests/foreman/ui/test_computeresource_gce.py:91]: json_key has multiple values to assert and our main params are zone and the whole key itself, the zone is getting asserted here hence thinking not to go for the whole assertion as if the data values are incorrect the CR will not be formed anyway.

File: tests/foreman/ui/test_computeresource_gce.py [hunk 2]
```diff
@@ -73,9 +77,7 @@ def test_positive_default_end_to_end_with_custom_profile(
                 'description': cr_description,
                 'provider': FOREMAN_PROVIDERS['google'],
                 'provider_content.http_proxy.value': http_proxy.name,
-                'provider_content.google_project_id': gce_cert['project_id'],
-                'provider_content.client_email': gce_cert['client_email'],
-                'provider_content.certificate_path': settings.gce.cert_path,
+                'provider_content.json_key': json_key,
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_computeresource_gce.py:80]: @adarshdubey-star there are some more tests in ui/test_host that will require this update, are you going to update them as well? 
OR consider migrating them under ui/test_computeresource_gce as it make more sense to keep those here, wdyt?

**Review by Gauravtalreja1 — CHANGES_REQUESTED** [review]

File: tests/foreman/ui/test_computeresource_gce.py (hunk 1 above)
Comment (shweta83) [reply tests/foreman/ui/test_computeresource_gce.py:91]: In that case, you can skip these assertion.

File: tests/foreman/ui/test_computeresource_gce.py (hunk 2 above)
Comment (shweta83) [reply tests/foreman/ui/test_computeresource_gce.py:80]: There is one in upgrades/test_host.py as well. Please update that test case too.

Comment (adarshdubey-star) [reply tests/foreman/ui/test_computeresource_gce.py:80]: I think we can migrate those test-cases in this module @Gauravtalreja1

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_computeresource_gce.py
airgun: 807

**Comment by adarshdubey-star** [comment]
I don't know why this build is failing.

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_computeresource_gce.py -k test_positive_default_end_to_end_with_custom_profile
airgun: 807

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_computeresource_gce.py -k test_positive_default_end_to_end_with_custom_profile
airgun: 807

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_computeresource_gce.py -k test_positive_default_end_to_end_with_custom_profile
airgun: 807
pod_resources_size: extra_large

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_computeresource_gce.py
airgun: 807
pod_resources_size: extra_large

**Comment by adarshdubey-star** [comment]
We can get this PR in, the migrated tests from the host module, which will be failing because of a known issue being worked on.

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_computeresource_gce.py -k test_positive_gce_provision_end_to_end
airgun: 811

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_computeresource_gce.py
airgun: 811

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_computeresource_gce.py
airgun: 811
pod_resources_size: extra_large

**Comment by adarshdubey-star** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_computeresource_gce.py
pod_resources_size: extra_large

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_computeresource_gce.py
pod_resources_size: extra_large

**Comment by adarshdubey-star** [comment]
The test case failing is not related to the change!

**Review by Gauravtalreja1 — APPROVED** [review]

**Comment by shweta83** [comment]
@adarshdubey-star There is one test failing in the PRT. Can you please check.

**Comment by adarshdubey-star** [comment]
> @adarshdubey-star There is one test failing in the PRT. Can you please check.

The failure is not related to the change something needs to be updated on the airgun side, I will check that later

**Review by shweta83 — APPROVED** [review]
Ack

**Review by shubhamsg199 — APPROVED** [review]
Ack

**Pull request merged**